applications/appunorganized/appunorganized-plugins/org.csstudio.swt.rtplot/src/org/csstudio/swt/rtplot/internal/Plot.java

```diff
@@ -243,7 +246,11 @@ else if (type == Instant.class)
         () ->
         {
             plot_processor.autoscale();
-            updateImageBuffer();
+            // In Eclipse 2020-12 we cannot seem to run updateImageBuffer() on
```

Comment:
This PR shouldn't really include these lines, they're the ones regarding the plot crashing and are handled elsewhere.

Reply:
Agreed, thanks for catching that. I have reverted this change in the latest commit.